src/features/AuthForm/components/LoginForm.tsx

```diff
@@ -0,0 +1,101 @@
+import { ChangeEvent, useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import Input from "shared/components/Input";
+import styles from "./AuthForm.module.css";
+import hideIcon from "assets/images/btn_hide.svg";
+import kakaoIcon from "assets/images/ic_kakao.svg";
+import googleIcon from "assets/images/ic_google.svg";
+import Button from "shared/components/Button";
+
+interface Values {
+  email: string;
+  password: string;
+}
+
+const INITIAL_VALUES = {
```

Comment:
P3:
관련된 type을 정의하셨으니 의도를 더 잘 나타낼 수 있게 아래처럼 쓰시는 것을 추천드립니다~

```suggestion
const INITIAL_VALUES: Values = {
```